tests/testament/tshould_not_work.nim

```diff
@@ -1,5 +1,6 @@
 discard """
 cmd: "testament/testament --directory:testament --colors:off --backendLogging:off --nim:$nim category shouldfail"
+targets: "c"
```

Comment:
```suggestion
```
those overrides should be reverted in the other affected tests too.

Reply:
it is intended, on cpp based platforms, ci failed

Reply:
fixed this file, other files are intended.